oidc-ui/src/pages/Login.js

```diff
@@ -21,45 +23,41 @@ import openIDConnectService from "../services/openIDConnectService";
 import DefaultError from "../components/DefaultError";
 import Password from "../components/Password";
 
-//authFactorComponentMapping
-const comp = {
-  PIN: Pin,
-  OTP: Otp,
-  BIO: L1Biometrics,
-  PWD: Password
-};
-
-function InitiateL1Biometrics(openIDConnectService) {
+function InitiateL1Biometrics(openIDConnectService, handleMoreWaysToSignIn) {
```

Comment:
can handleMoreWaysToSignIn method be renamed to handleBackButtonClick

Reply:
resolve in this [f23c62b](https://github.com/mosip/esignet/pull/367/commits/f23c62bca823f172be40fc5ad7a41974c9ebf45b)